src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -22,7 +21,7 @@
 
     private final Optional<BibDatabase> database;
 
-    private final LayoutFormatter toUnicode = new LatexToUnicodeFormatter();
+    private final LatexToUnicodeFormatter toUnicode = new LatexToUnicodeFormatter();
```

Comment:
Is there a particular reason for using the exact type in the variable declaration? Why not just `Formatter`?